e2e/actions/l2_challenger_test.go

```diff
@@ -351,7 +573,7 @@ interaction:
 			txHash = validator.ActBisect(t, outputIndex)
 			includeL1Block(t, miner, validator.address)
 		case chal.StatusAsserterTimeout, chal.StatusReadyToProve:
-			txHash = challenger.ActProveFault(t, outputIndex)
+			txHash = challenger.ActProveFault(t, outputIndex, true)
 			includeL1Block(t, miner, challenger.address)
```

Comment:
L575-L577 should be like below. Currently the test fails.
```
		case chal.StatusAsserterTimeout:
			// not expected
		case chal.StatusReadyToProve:
			txHash = challenger.ActProveFault(t, outputIndex, false)
			includeL1Block(t, miner, challenger.address)
```

Reply:
Good point.
It should be false :)